Rakefile

```diff
@@ -134,7 +134,9 @@ case @osfamily
 end
 
 # The Puppet 4 load path
-@p4libdir = "/opt/puppetlabs/puppet/lib/ruby/vendor_ruby"
+@p4home = "/opt/puppetlabs/puppet"
```

Comment:
@rbrw @johnduarte the values look sane. It looks like this script only needs to run `puppet` so using `/opt/puppetlabs/bin` should be fine.